src/qtxdg/xdgdesktopfile.cpp

```diff
@@ -986,7 +986,7 @@ static QStringList parseCombinedArgString(const QString &program)
 void replaceVar(QString &str, const QString &varName, const QString &after)
 {
     str.replace(QRegExp(QString::fromLatin1("\\$%1(?!\\w)").arg(varName)), after);
-    str.replace(QRegExp(QString::fromLatin1("\\$\\{%1\\}").arg(varName)), after);
+    str.replace(QRegExp(QString::fromLatin1(R"(\$\{%1\})").arg(varName)), after);
```

Comment:
...and why not in the line above? :)

Reply:
clang-tidy, probably, got confused by the parentheses.
I only checked what it changed.

``` cpp
R"regexp(\$%1(?!\w))regexp"
```
cuts it.

Will add ASAP.